manim/mobject/coordinate_systems.py

```diff
@@ -495,3 +503,321 @@ def get_coordinate_labels(self, *numbers, **kwargs):
     def add_coordinates(self, *numbers):
         self.add(self.get_coordinate_labels(*numbers))
         return self
+
+
+class PolarPlane(NumberPlane):
+    """A version of `NumberPlane` for polar coordinates.
+
+    Parameters
+    ----------
+    azimuth_line_frequency : :class:`float`, optional
+        The frequency of faded lines in the azimuth, expressed in units of revolution
+        (so :math:`\\frac{1}{n}` would result in :math:`n` lines). If ``None`` is specified then it will use the default
+        specified by ``azimuth_units``:
+
+            - ``"PI radians"`` or ``"TAU radians"``: :math:`\\frac{1}{20}`
+            - ``"degrees"``: :math:`\\frac{1}{36}`
+            - ``"gradians"``: :math:`\\frac{1}{40}`
+            - ``None``: :math:`1`
+
+    radius_line_frequency : :class:`float`, optional
+        The frequency of faded lines in the radius.
+
+    azimuth_units : Optional[:class:`str`], optional
+        Specifies a default labelling system for the azimuth. Choices are:
+
+            - ``"PI radians"``: Fractional labels in the interval :math:`\\left[0, 2\\pi\\right]` with :math:`\\pi` as a constant.
+
+              .. manim:: PolarPlanePIradians
```

Comment:
I think it would be less disruptive for the documentation if there was one rendered example illustrating all four choices for `azimuth_units`.